pkg/blobstore/local/flat_blob_access.go

```diff
@@ -351,8 +397,10 @@ func (ba *flatBlobAccess) FindMissing(ctx context.Context, digests digest.Set) (
 	// one thread.
 	ba.refreshLock.Lock()
 	defer ba.refreshLock.Unlock()
-
+	// Add refresh start time before the refresh loop
+	refreshStart := time.Now()
 	blobsRefreshedSuccessfully := 0
+	blobRefreshSizeBytes := 0
```

Comment:
Please change this to `int64`, so that the `int()` conversion isn't needed. It's fairly easy to overflow `int` on 32-bit architectures.